src/app/core/services/github.service.ts

```diff
@@ -233,6 +233,26 @@ export class GithubService {
     octokit.issues.updateLabel({owner: ORG_NAME, repo: REPO, name: labelName, current_name: labelName, color: labelColor});
   }
 
+  /**
+   * Checks if the given list of users are allowed to be assigned to an issue.
+   * Throws an error if at least one of them is not allowed.
+   * @param assignees - GitHub usernames to be checked
+   */
+  areUsersAssignable(assignees: string[]): Observable<void> {
+    return forkJoin(assignees.map(a => this.isUserAssignable(a))).pipe(map(x => void 0));
+  }
+
+  isUserAssignable(assignee: string): Observable<boolean> {
+    return from(octokit.issues.checkAssignee({
+      owner: ORG_NAME,
+      repo: REPO,
+      assignee: assignee
+    })).pipe(
+      every((response: any) => response.status === 204),
+      catchError(_ => throwError(`Cannot assign ${assignee} to the issue. Please check if ${assignee} is authorized.`))
+    );
+  }
```

Comment:
I realise it would be better to reduce the calls to GitHub by requesting list of assignable users and performing the check locally. This would probably reduce the chance of getting a "Too many requests" error

Reply:
That's a valid point. Also, if there are duplicates of the issue, then the call to check if the users are assignable would need to be performed multiple times. I will change the code such that only 1 request to check for assignees is sent.